types/wlr_foreign_toplevel_management_v1.c

```diff
@@ -405,6 +405,36 @@ void wlr_foreign_toplevel_handle_v1_set_fullscreen(
 	toplevel_send_state(toplevel);
 }
 
+static void send_parent(struct wlr_foreign_toplevel_handle_v1 *parent,
+		struct wl_resource *toplevel_resource) {
```

Comment:
The order of these arguments is a little bit misleading, I'd prefer having the `wl_resource` first and then the parent. And maybe we can replace the `wl_resource` argument with a proper `wlr_foreign_toplevel_handle_v1`, to avoid mixing resource types up.